tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,171 @@
+            time_between_pvcs_deletion_success))
+        # Verify space has been reclaimed
+        used_after_deletion = self.ceph_cluster.check_ceph_pool_used_space(
+            self.ceph_cluster.cluster_name)
+        log.info("Verifying space has been reclaimed...")
+        assert used_after_deletion < used_before_deletion, (
+            f"Space hasn't been reclaimed after PVs deletion."
+            f"/nUsed space before deletion: " f"{used_before_deletion}"
+            f"/nUsed space after deletion: {used_after_deletion}"
+        )
+        log.info("Space reclaimed successfully.")
+
+        # Project 1 done, moving on to Project 2
+        log.info("Creating Project 2")
+        project_2 = project_factory()
+
+        log.info("Creating a mix of {} CephFS & RBD PVCs "
+                 "(each bound to an app pod)".format(2 * pvcs_num))
+        # Create CephFS PVCs bound to an app pod
+        for pvc_obj in multi_pvc_factory(interface=constants.CEPHFILESYSTEM,
+                                         project=project_2,
+                                         num_of_pvc=pvcs_num):
+            pod_factory(pvc=pvc_obj, interface=constants.CEPHFILESYSTEM)
+        # Create RBD PVCs bound to an app pod
+        for pvc_obj in multi_pvc_factory(interface=constants.CEPHBLOCKPOOL,
+                                         project=project_2,
+                                         num_of_pvc=pvcs_num):
+            pod_factory(pvc=pvc_obj, interface=constants.CEPHBLOCKPOOL)
+
+        # Delete the entire Project 2 (along with all of its PVCs)
+        pvs = helpers.get_all_pvs()
+        used_before_deletion = self.ceph_cluster.check_ceph_pool_used_space(
+            self.ceph_cluster.cluster_name)
+        start_time = time.time()
+        # Switch back to default project
+        ocp.switch_to_default_rook_cluster_project()
+        log.info("Deleting Project 2")
+        project_2.delete(resource_name=project_2.namespace)
+        log.info("Waiting for Project 2 deletion success...")
+        project_2.wait_for_delete(resource_name=project_2.namespace)
+        log.info("Project 2 deleted successfully.")
+        project_deletion_time = time.time() - start_time
+        log.info(
+            "Project 2 deletion time: {} seconds".format(project_deletion_time))
+        # PVs delete might still be running, check until all PVs are deleted
+        # Check time between first PV delete and last PV delete
+        time_between_pvcs_deletion_success = 0
+        # Also check delete time per each PV
+        individual_pv_delete_times = []
+        for pv in pvs:
+            pv_start_time = time.time()
+            pv_name = pv.backed_pv
+            pv.ocp.wait_for_delete(pv_name)
+            helpers.validate_pv_delete(pv_name)
+            pv_delete_time = time.time() - pv_start_time
+            individual_pv_delete_times.append({pv_name: pv_delete_time})
+            time_between_pvcs_deletion_success += pv_delete_time
```

Comment:
time_between_pvcs_deletion_success += pv_delete_time

This value will be appropriate if it is the time difference  before and after this loop.

Reply:
@jilju So basically move it outside the loop?